### ISSUE TITLE
Support TF 0.15.0 syntax

### ISSUE THREAD

**Issue by wagnst** [terraform-linters/tflint#1098]
Latest 0.15 TF introduced additional parameter in `required_providers` block: https://github.com/hashicorp/terraform/blob/v0.15/CHANGELOG.md#0150-april-14-2021

If running latest `tflint` it gives:

```
Error: 3 loader.go:85: [ERROR] versions.tf:6,32-37: Variables not allowed; Variables may not be used here., and 2 other diagnostic(s)
Failed to load configurations. 3 error(s) occurred:

Error: Variables not allowed

  on versions.tf line 6, in terraform:
   6:       configuration_aliases = [vault.xxx, vault.yyy]

Variables may not be used here.

Error: Variables not allowed

  on versions.tf line 6, in terraform:
   6:       configuration_aliases = [vault.xxx, vault.yyy]

Variables may not be used here.

Error: Invalid required_providers object

  on versions.tf line 3, in terraform:
   3:     vault = {
   4:       source                = "hashicorp/vault"
   5:       version               = "~> 2.19.0"
   6:       configuration_aliases = [vault.xxx, vault.yyy]
   7:     }

required_providers objects can only contain "version" and "source" attributes. To configure a provider, use a "provider" block.

Error: Process completed with exit code 2.
```

TF Code

```hcl
terraform {
  required_providers {
    vault = {
      source                = "hashicorp/vault"
      version               = "~> 2.19.0"
      configuration_aliases = [vault.xxx, vault.yyy]
    }
}
```

GitHub Actions
```yaml
- uses: terraform-linters/setup-tflint@v1
  name: Setup TFLint
  with:
    tflint_version: latest
    github_token: ${{ secrets.GITHUB_TOKEN }}

- shell: bash
  name: tflint
  run: |
    TFLINT_LOG=info tflint --module -f compact
```

**Comment by bendrucker** [comment]
https://github.com/hashicorp/terraform/issues/26418 has not happened yet so we can do this easily. See https://github.com/terraform-linters/tflint/issues/937 for more background.
